client/src/components/MarketUpdate/UpdateTurnipSelectors.js

```diff
@@ -0,0 +1,24 @@
+export function createTurnipPayload(enteredTurnipData, islandId) {
+	const { date, timeOfDay, price } = enteredTurnipData;
+	const year = date.getFullYear();
+	const month = date.getMonth();
+	const day = date.getDate();
+	const time = timeOfDay === "am" ? 1 : 13;
```

Comment:
maybe we should just use 0 and 12? That's the real timestamp that the markets change at

Reply:
changed